CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+2.0.2 - 2017-07-27
+~~~~~~~~~~~~~~~~~~
+
+* Marked all symbols as hidden in the ``manylinux1`` wheel to avoid a
+  bug with symbol resolution in older Linux distributions.
```

Comment:
The bug isn't with older Linux distributions, it just happens to materialize with a python shipped by an older distro.

I don't know what the right thing to write here is.